### ISSUE TITLE
`application:toggle-side-tabbar` does not work as expected

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/jupyterlab#15760]
## Description

The sidebar shortcuts introduced in https://github.com/jupyterlab/jupyterlab/pull/14799 do not open individual sidebar items but just collapse/uncollapse the tabbar. This is very different from the approach that I pushed in https://github.com/jupyterlab/jupyterlab/pull/14799/commits/6c3a552bdc2580813ea82861b2bbb6b21f1a82eb and I do not understand why such an approach was finally taken (@g547315, @brichet?)

## Reproduce

![does-not-work](https://github.com/jupyterlab/jupyterlab/assets/5832902/1c7769e2-50d0-4fbe-831a-5bee87f5befe)

## Expected behavior

- <kbd>Alt</kbd> + <kbd>1</kbd> opens first element in left sidebar (e.g. file browser)
- <kbd>Alt</kbd> + <kbd>2</kbd> opens second element in left sidebar (e.g. running kernels)
- etc...

## Context

- JupyterLab version: 4.1.0

**Comment by yingding** [comment]
Same issue here on a German Mac Keyboard, the combo
"opt + 8" or "opt + 9" is used for "{" and "}", now I can not type dictionary in python anymore.

It just toggle-side-tabbar. How can I deactivate this toggle in jupyterlab. Please don't add any more short keys.

**Comment by krassowski** [comment]
@yingding the issue with typing on international keyabords on Mac is tracked in https://github.com/jupyterlab/jupyterlab/issues/15744 and will be fixed in https://github.com/jupyterlab/jupyterlab/pull/15761. This issue is about a command not working as expected.

PS. in the future feel welcome to provide an early feedback during beta testing period :)

**Comment by yingding** [comment]
thanks, sorry for the confusion.

**Comment by brichet** [comment]
@krassowski indeed the intention of the PR was to toggle the sidebar widgets and not the tabbar.

I'm not able to test it right now, I'll look at it next week.

I wonder if this is not related to the selector used to retrieve the current active widget.

**Comment by ammannc** [comment]
## This makes using jupyterlab with a german keyboard impossible

..since you can no longer type square and curved brackets.

The fix (see below) is also annoying as you have to manually disable each shortcut in the JSON settings. (also makes me wonder why there is no easier option to disable a default shortcut in the settings)

Wha are there so many default shortcuts for this function:

<img width="990" alt="Bildschirmfoto 2024-02-08 um 15 48 05" src="https://github.com/jupyterlab/jupyterlab/assets/92221670/ca2321e4-6ae7-43ca-a460-0cbe82b1a5f4">

## Fix for german keyboards

```
{
    "shortcuts": [
        {
            "args": {
                "side": "right",
                "index": 0
            },
            "command": "application:toggle-side-tabbar",
            "keys": [
                "Alt 5"
            ],
            "selector": "body",
            "disabled": true
        },
        {
            "args": {
                "side": "right",
                "index": 1
            },
            "command": "application:toggle-side-tabbar",
            "keys": [
                "Alt 6"
            ],
            "selector": "body",
            "disabled": true
        },
        {
            "args": {
                "side": "right",
                "index": 2
            },
            "command": "application:toggle-side-tabbar",
            "keys": [
                "Alt 8"
            ],
            "selector": "body",
            "disabled": true
        },
        {
            "args": {
                "side": "right",
                "index": 3
            },
            "command": "application:toggle-side-tabbar",
            "keys": [
                "Alt 9"
            ],
            "selector": "body",
            "disabled": true
        }
    ]
}
```

**Comment by krassowski** [comment]
@ammannc I understand. As mentioned before this issue is tracked separately in https://github.com/jupyterlab/jupyterlab/issues/15744 and a pull request with a fix is waiting for review in https://github.com/jupyterlab/jupyterlab/pull/15761. Are you able to help with review of the pull request so that we can merge and release the fix sooner?

**Comment by g547315** [comment]
Thanks for reporting the issue and bringing this to my attention.

You're right, the current behavior doesn't work as expected, I'm already working on a fix for this bug and will open a PR shortly.

**Comment by brichet** [comment]
@g547315 just saw your PR...

**Comment by krassowski** [comment]
Closing as fixed by #15791
